instance/models/instance.py

```diff
@@ -218,3 +223,14 @@ def delete(self, *args, **kwargs):
         if not kwargs.pop('ref_already_deleted', False):
             self.ref.delete(instance_already_deleted=True)
         super().delete(*args, **kwargs)
+
+
+class InstanceTag(ValidateModelMixin, models.Model):
+    """
+    Custom tags that can be applied to instances, for filtering and annotation.
+    """
+    name = models.SlugField(max_length=100, blank=False, unique=True)
```

Comment:
@itsjeyd I just couldn't remember the default value, and couldn't be bothered to look it up.  I'll remove it.